generators/contract_report.py

```
@@ -0,0 +1,334 @@
+"""
+import sys
+import os
+import logging
+from typing import List, Dict, Any, Optional
+from datetime import datetime
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+# 내부 모듈 - 기존 LLM 코드에 맞게 import
+from model.clause_checker import (
+    ContractLegalChecker, 
```

Ruff (0.12.2)

`model.clause_checker.ContractLegalChecker` imported but unused

Remove unused import: `model.clause_checker.ContractLegalChecker`

(F401)